forms/collections/pages.js

```diff
@@ -35,12 +41,6 @@ define(function (require) {
       view.render();
       currentform.attributes._view.$el.append(view.el);
 
-      this.forEach(function (page, number) {
-        if (number !== index) {
-          page.removeView();
-        }
-      });
-
```

Comment:
This is the "delete other page views" loop I mentioned above. This was handy in case errant API use somehow created more page views than we needed. I'm pretty sure our own code doesn't do this though, so maybe this code was unnecessarily cautious. /shrug